opl/opl_sdl.c

```diff
@@ -173,8 +173,8 @@ static void FillBuffer(int16_t *buffer, unsigned int nsamples)
 
         for (i=0; i<nsamples; ++i)
         {
-            buffer[i * 2] = (int16_t) mix_buffer[i * 2];
-            buffer[i * 2 + 1] = (int16_t) mix_buffer[i * 2 + 1];
+            buffer[i * 2 + 1] = (int16_t) mix_buffer[i * 2];
```

Comment:
I reversed the values from  that pan code myself when I was diagnosing the problem and noticed no change on some music and odd panning on others.

From zdoom OPL source:
```cpp
void Channel::updatePan(OPL3 *OPL3) {
	if (!OPL3->FullPan)
	{
		if (OPL3->_new == 0)
		{
			leftPan = VOLUME_MUL;
			rightPan = VOLUME_MUL;
		}
		else
		{
			leftPan = cha * VOLUME_MUL;
			rightPan = chb * VOLUME_MUL;
		}
	}
}
```
ZDoom assumes CHA is left. Odd.